doc/start.md

```diff
@@ -96,3 +96,28 @@ Below is the same diagram as above, except with the options
 Δ child=undefined   <-  parent=undefined   =>  child=undefined  parent=undefined  None
 Δ child=3           <-  parent=3           =>  child=3          parent=3          None
 ```
+
+The `onInitDoNotUpdateParent` option can change how initialization works. This
+option’s value is `false` by default, but if it’s set to `true`, then the parent
+will _never_ be set when the binding is initialized. This option does not effect
+one-way parent-to-child bindings because the child’s value is never set when
+that type of binding is initialized.
+
+Below is the same diagram as above, except with the options
```

Comment:
The “above” diagram is the `onInitDoNotUpdateChild=true` one, so maybe this should say something like “Below is a diagram that shows the change (Δ) from the default behavior when `onInitDoNotUpdateParent=true`”.

Also, the diagram should have the Δ column and highlight which bindings changed.